daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -351,6 +351,48 @@ async function start() {
 }
 ```
 
+#### Bulk Subscribe to messages
+
+Bulk Subscription is supported and is available through following API:
+
+- Bulk subscription through the `subscribeBulk` method: `maxMessagesCount` and `maxAwaitDurationMs` are optional; and if not provided, default values for related components will be used.
+
+App need not iterate through multiple entries inside a received bulk subscribe event. Callback provided by App is applicable per Entry. And, similarly, while writing callback, App can decide to send back SUCCESS/RETRY/DROP, as defined above.
```

Comment:
While listening for messages, the application receives messages from Dapr in bulk. However, like regular subscribe, the callback function receives a single message at a time, and the user can choose to return a `DaprPubSubStatusEnum` value to acknowledge successfully, retry, or drop the message. The default behavior is to return a success response.